rollup/cmd/mock_app.go

```diff
@@ -27,9 +30,8 @@ type MockApp struct {
 	args []string
 }
 
-// NewRollupApp return a new rollupApp manager, name mush be one them.
+// NewRollupApp TODO function will be replaced by NewRollupApp2
 func NewRollupApp(base *docker.App, file string) *MockApp {
```

Comment:
could we keep this and only use `testcontainers`? Is there any blocker here?

Reply:
It is still needed now, and some docker has not been completely replaced. I will submit a few more PRs later.